force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +446,11 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if(person1.getAge() >= person2.getAge())
+        {return person1;}
+        else {
+            return person2;
+        }
```

Comment:
Your curly braces around the if statement should be set up like this (and don't forget to indent):
```suggestion
        if(person1.getAge() >= person2.getAge()){
           return person1;
        } else {
            return person2;
        }
```